docs/app-dev-context/features.md

```diff
@@ -8,4 +10,14 @@ If you can't find what you are looking for in the Application development guide,
 
 :::
 
-This section covers the many features that are available to use in your [Temporal Application](/concepts/what-is-a-temporal-application).
+In this section you will find the following:
```

Comment:
[here and in Foundations] To avoid using the present tense (as well as the implication that the reader _will_ find the topics, by gum), perhaps use one of the following:

- In this section you can find the following:
- This section contains the following how-to pages:
- In this section:

Reply:
Whoops.
Ya, I think "In this section you can find the following:" fits our style guidance.